refiner-lib/src/refiner_inner.rs

```diff
@@ -934,6 +996,46 @@ fn build_transaction(
     })
 }
 
+fn determine_ft_on_transfer_addresses(
+    execution_outcome: &ExecutionOutcomeWithReceipt,
+    args: &NEP141FtOnTransferArgs,
+    storage: &Storage,
+    near_block: &BlockView,
+    transaction_index: u32,
+) -> (Address, Option<Address>) {
+    // For now, we use `aurora` as the `from address` for both ETH and ERC-20 tokens.
+    // Later, when the engine and ethconnector split is deployed on mainnet,
+    // this will be changed to the address of ethconnector.
+    let from_address = near_account_to_evm_address(b"aurora");
```

Comment:
`aurora` should be replaced with the value from `storage.get_engine_account_id()`

Reply:
done